mkdocs.yml

```diff
@@ -90,6 +90,8 @@ nav:
         - Visit: polaris/visualization/visit.md
         - FFmpeg: polaris/visualization/ffmpeg.md
         - ImageMagick: polaris/visualization/imagemagick.md
+        - Paraview tutorial: polaris/visualization/paraview-tutorial.md
```

Comment:
Looks like it should be capitalized as "ParaView" everywhere?

Reply:
Yes.  And we should ensure that "VisIt" is consistent, as well.

Reply:
Updated now. Thanks for the feedback